### PR TITLE
Update kafka reference doc

### PR DESCRIPTION
**Pull request by hui1110**
Sync the following PR:

- https://github.com/MicrosoftDocs/azure-dev-docs-pr/pull/3406

### CODE REVIEW

File: docs/src/main/asciidoc/kafka-support.adoc [hunk 1]
```diff
@@ -233,7 +236,7 @@ spring.cloud.stream.kafka.binder.brokers=<NAMESPACENAME>.servicebus.windows.net:
 spring.cloud.stream.binders.kafka.environment.spring.main.sources=com.azure.spring.cloud.autoconfigure.kafka.AzureKafkaSpringCloudStreamConfiguration
 ----
 
-NOTE: Set `com.azure.spring.cloud.autoconfigure.kafka.AzureKafkaSpringCloudStreamConfiguration` to enable the whole OAuth authentication workflow.
+NOTE: If you're using version `4.3.0`, don't forget to set the `spring.cloud.stream.binders.<kafka-binder-name>.environment.spring.main.sources=com.azure.spring.cloud.autoconfigure.kafka.AzureKafkaSpringCloudStreamConfiguration` property to enable the whole OAuth authentication workflow, where `kafka-binder-name` is `kafka` by default in a single Kafka binder application. The configuration `AzureKafkaSpringCloudStreamConfiguration` specifies the OAuth security parameters for `KafkaBinderConfigurationProperties`, which is used in `KafkaOAuth2AuthenticateCallbackHandler` to enable Azure Identity. For version after `4.4.0`, this property will be added automatically for each Kafka binder environment, so there's no need for you to add it manually.
```
Comment (yiliuTo) [inline docs/src/main/asciidoc/kafka-support.adoc:239]: We can place 
```
For version after `4.4.0`, this property will be added automatically for each Kafka binder environment, so there's no need for you to add it manually.
```
 in a new line.

**Review by yiliuTo — DISMISSED** [review]
LGTM

**Review by yiliuTo — APPROVED** [review]
LGTM

File: docs/src/main/asciidoc/kafka-support.adoc [hunk 2]
```diff
@@ -236,7 +236,7 @@ spring.cloud.stream.kafka.binder.brokers=<NAMESPACENAME>.servicebus.windows.net:
...
+NOTE: If you're using version `4.3.0`, don't forget to set the `spring.cloud.stream.binders.<kafka-binder-name>.environment.spring.main.sources=com.azure.spring.cloud.autoconfigure.kafka.AzureKafkaSpringCloudStreamConfiguration` property to enable the whole OAuth authentication workflow, where `kafka-binder-name` is `kafka` by default in a single Kafka binder application. The configuration `AzureKafkaSpringCloudStreamConfiguration` specifies the OAuth security parameters for `KafkaBinderConfigurationProperties`, which is used in `KafkaOAuth2AuthenticateCallbackHandler` to enable Azure Identity. For version after `4.4.0`, this property will be added automatically for each Kafka binder environment, so there's no need for you to add it manually..
```
Comment (saragluna) [inline docs/src/main/asciidoc/kafka-support.adoc:239]: Extra `.` at the end of the line.

**Pull request merged**